roomba/entry_points.py

```diff
@@ -0,0 +1,35 @@
+import sys
+from roomba.discovery import RoombaDiscovery
+from roomba.getpassword import RoombaPassword
+
+
+def discovery():
+    print(_get_roomba_info())
+
+
+def password():
```

Comment:
```
def password():
    if len(sys.argv) == 1:
        roomba_info = _get_roomba_info()
        roomba_ip = roomba_info.ip
    else:
        roomba_ip = str(sys.argv[1])
    _wait_for_input()
    print(_get_roomba_password(roomba_ip))
```
I had to do something similar for password as well as I kept having it try the wrong it saw first